### PR TITLE
Use indico instead of a google form to register as participant

### PR DESCRIPTION
**Pull request by Chilipp**
As discussed quite some time ago on slack, we'd like to use indico for the registration of participants for an event. Therefore I updated the [privacy policy][privacy], as well as the FAQs about [how to attend an event][attend] and [registration and fees][fees].

@ClaireWyatt and @eileen-kuehn, did I forget something?

@eileen-kuehn: in the privacy policy, I linked to the privacy policy of our SORSE event. Do I need to use something different here?

[privacy]: https://554-267395254-gh.circle-artifacts.com/0/SORSE/privacy/index.html
[attend]: https://554-267395254-gh.circle-artifacts.com/0/SORSE/faq/howto/attendance.html
[fees]: https://554-267395254-gh.circle-artifacts.com/0/SORSE/faq/about/costs.html

### CODE REVIEW

**Comment by eileen-kuehn** [comment]
@Chilipp, the privacy policy in the current Indico event does not cover registration yet. We still need to provide a privacy policy for event registration (I already have a version we can adapt for our purposes). So we should probably add the registration section to the "main event" (the event for abstract submissions). However, we will also have privacy policies with each other event we create for registration. But I doubt you want to link those.

File: _faq/about/03-costs.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ tags:
 ---
 ## Registration
 
-Registration for each event will be via a simple Google Form owned by Soc-RSE. All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors. 
+Registration for each event will be via dedicated registration link on its webpage.
```
Comment (eileen-kuehn) [inline _faq/about/03-costs.md:9]: It does not become clear what *on its webpage* means. Do you refer to the page advertising the contribution on the website or do you mean the event pages in indico?

File: _faq/about/03-costs.md [hunk 2]
```diff
@@ -6,8 +6,12 @@ tags:
...
+All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors.
```
Comment (eileen-kuehn) [inline _faq/about/03-costs.md:10]:
```suggestion
All data will be deleted from Indico after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors.
```

Please note, that we cannot promise deletion of all information as we cannot guarantee what happens to the data that was forwarded to sponsors.

File: _faq/about/03-costs.md [hunk 3]
```diff
@@ -6,8 +6,12 @@ tags:
...
 
 ## Do I need to pay to attend SORSE events?
 
-No! It’s completely free to the community with the national RSE Groups from many countries collaborating to bring this to you. 
+No! It’s completely free to the community with the national RSE Groups from many countries collaborating to bring this to you.
+
+
+[indico]: https://indico.scc.kit.edu
```
Comment (eileen-kuehn) [inline _faq/about/03-costs.md:17]: I propose additionally linking to the category where we will create all the events in, so that visitors can see what is going to happen when.
You can find it at https://indico.scc.kit.edu/category/95/

File: _faq/howto/attendance.md [hunk 4]
```diff
@@ -4,8 +4,9 @@ tags:
   - How it works
 ---
 
-Registration for each event will be via a simple Google Form owned by [Soc-RSE]({% include fix-link.html link=site.data.committee.ukRSE.internal %}) (see [Registration and Fees]({% include fix-link.html link="/faq/about/costs" %})).
+Every event has a registration link on its webpage where you can sign up in order
```
Comment (eileen-kuehn) [inline _faq/howto/attendance.md:7]:
```suggestion
Every event has a registration link on its webpage where you can sign up
```

File: _pages/privacy.md [hunk 5]
```diff
@@ -39,9 +39,10 @@ toc_sticky: true
 
 - We use a Google form to submit to the topic bazaar, but this is also stated
   separately [here]({% include fix-link.html link=submit_topic %}).
-- Participants register using a Google form for events.
+- Participants register via the [indico](https://indico.scc.kit.edu) platform
+  ([privacy statement](https://indico.scc.kit.edu/event/863/page/550-privacy-policy))
```
Comment (Chilipp) [inline _pages/privacy.md:43]:
```suggestion
- Participants register via the [indico](https://indico.scc.kit.edu/category/95/) platform
  using a dedicated registration link.
```
see https://github.com/SORSE/sorse.github.io/pull/229#issuecomment-659245451

Then let's remove the link to the privacy policy here

File: _faq/about/03-costs.md (hunk 1 above)
Comment (Chilipp) [reply _faq/about/03-costs.md:9]: it's the webpage of the event at sorse.github.io, http://www.philipp-s-sommer.de/sorse.github.io/programme/software_demos/event-001/ for instance. any ideas how to rephrase it to make it more clear @eileen-kuehn?

File: _faq/about/03-costs.md (hunk 3 above)
Comment (Chilipp) [reply _faq/about/03-costs.md:17]:
```suggestion
```
actually this should be removed, as it's not used anywhere in the text :sweat_smile: 
I added the link to the privacy policy (see https://github.com/SORSE/sorse.github.io/pull/229/files#r455623527). Or shall we add it to the text here as well?

File: _faq/about/03-costs.md (hunk 1 above)
Comment (eileen-kuehn) [reply _faq/about/03-costs.md:9]: You might simply say the following.

```suggestion
Registration for each event will be via dedicated registration in Indico.
```

To ensure that the following sentence properly describes the data you could then go on with "All registration data will be deleted [...]".

File: _faq/about/03-costs.md (hunk 3 above)
Comment (eileen-kuehn) [reply _faq/about/03-costs.md:17]: I think it is sufficient to have the proper link in the privacy policies.

File: _faq/about/03-costs.md (hunk 1 above)
Comment (Chilipp) [reply _faq/about/03-costs.md:9]: Sounds good. But I think we should still tell them _where_ to find this registration link. Maybe something like
```suggestion
Registration for each event will be via dedicated registration in Indico.
Simply select the event from the [events calendar](){: .missing} and click the 
_Register_ button on its webpage.
```

Comment (eileen-kuehn) [reply _faq/about/03-costs.md:9]: Great! I like your solution. 
I only compared to the previous version where we also didn't tell them where to go O:)

What I just realised. The first sentence somehow reads strange (maybe just for me as a German?). We could further improve by adding *available*.

```suggestion
Registration for each event will be available via dedicated registration in Indico.
Simply select the event from the [events calendar](){: .missing} and click the 
_Register_ button.
```

Comment (Chilipp) [reply _faq/about/03-costs.md:9]: Sounds good! What about

```suggestion
Registration for each event will be available via Indico. Select the event from the 
[events calendar](){: .missing} and click the _Register_ button on its webpage.
```

I think that's straight-forward to understand

Comment (eileen-kuehn) [reply _faq/about/03-costs.md:9]: And another one skipping *on its webpage* :) I think click the button is sufficient.

```suggestion
Registration for each event will be available via Indico. Select the event from the 
[events calendar](){: .missing} and click the _Register_ button.
```

**Comment by Chilipp** [comment]
alright, thanks @eileen-kuehn! I'll make a small change in the tools FAQ as you proposed in https://github.com/SORSE/sorse.github.io/pull/229#pullrequestreview-449628660, one sec

**Comment by Chilipp** [comment]
alright, done @eileen-kuehn. See 3f51d47 or https://565-267395254-gh.circle-artifacts.com/0/SORSE/faq/howto/tools.html
